### DISCUSSION TITLE
Feature to select which domain to be default

### DISCUSSION THREAD

**Discussion by astronyu** [shlinkio/discussions#136] [Help wanted] [answered]
### Summary

I was wondering if there is already a way to set which domain to be default. I can see that both domain appears on my server but does not have the feature to be able to select as default. It just defaulted to whichever domain the script first installed

### Use case

I'm ceasing the use of an old domain and want to use the new domain without reinstalling the whole thing.

**Comment by acelaya** [comment, marked as answer]
This can already be done:

* If you are using env vars to configure Shlink, just provide a different value for `DEFAULT_DOMAIN`.
* If you configured Shlink via [installation tool](https://shlink.io/documentation/command-line-interface/installation-tool/), you can run `vendor/bin/shlink-installer set-option` in order to [update any configuration option](https://shlink.io/documentation/command-line-interface/installation-tool/#set-option), including the default domain.
